cloud/scope/powervs_cluster.go

```diff
@@ -847,18 +859,32 @@ func (s *PowerVSClusterScope) createServiceInstance() (*resourcecontrollerv2.Res
 	return serviceInstance, nil
 }
 
-// ReconcileNetwork reconciles network.
+// ReconcileNetwork reconciles network
+// If only IBMPowerVSCluster.Spec.Network is set, network would be validated and if exists already will get used as cluster’s network or a new network will be created via DHCP service.
```

Comment:
Can you also mention what happens when both network and dhcpserver is not set, Currently that configuration is used the most.